### ISSUE TITLE
Add certain dropdown fields to Metadata section of Admin for easy editing

### ISSUE THREAD

**Issue by monika-aashe** [AASHE/hub#159]
We would like to add the following fields to the METADATA FOR CONTENT TYPES section in Admin, because we anticipate staff needing to add to and change these in the future. This has already been done for some of the more commonly used fields such as Sustainability Topic and Discipline, and has also been done for the Program type field under Academic Programs. See http://aashe-hub-dev.herokuapp.com/admin/metadata/ 

Conference Presentations:
- Conference Name (current options: AASHE, Other)
- Presentation Type (current options: Poster, Presentation)

Course Materials:
- Type of Material (current options: Assignment or Exercise, Syllabus, Course Presentation)

Outreach Materials:
- Type of Material (logo, signs/poster, sticker, flyer/brochure, guide, map, other)

Publications:
- Type of Material (current options: book, book chapter, journal article, news or magazine article, blog article, published report, undergraduate student research, graduate student research)

**Comment by jamstooks** [comment]
Can we push this back a few phases? It doesn't feel like a requirement for launch...

**Comment by monika-aashe** [comment]
Type of Material (current options: book, book chapter, journal article, news or magazine article, blog article, published report, student thesis/dissertation, other student research) for Publications is especially important, since it's tied to Awards and there's no way for Crystal and I to determine how many awards submissions we have unless we click on all 60+ recently submitted publications. Changing to High Priority for this (all of the other ones are pretty low though so once this is done we can change back to prio:low.

**Comment by juliandsmith** [comment]
@monika-aashe This is just about being able to edit the options via the meta-data menu. It doesn't address the filter issue which is covered by a different high priority ticket. I'm going to switch this back to low priority and delete our comments to minimize confusion.

**Comment by jamstooks** [comment]
@monika-aashe - I have merged Scott's pull-request and loaded it on a dev server:

https://aashe-hub-dev-ben.herokuapp.com/

You can edit metadata here:

https://aashe-hub-dev-ben.herokuapp.com/_ad/metadata/

Would you mind just double-checking that this is working as desired and then I will merge it into production...

**Comment by monika-aashe** [comment]
It looks great to me, thanks!
